### DISCUSSION TITLE
replit架设的alist如何自己修复，求大佬指点

### DISCUSSION THREAD

**Discussion by KJFKA** [AlistGo/alist#3443] [General]
replit架设的alist如何自己修复，求大佬指点

**Comment by ywjzzz** [comment]
同问 应该升级一下就OK了吧

**Reply by KJFKA** [reply]
关键是咋升级？我只知道PC架设的alist直接下载最新版本替换就行，replit搭建的我不会升级呜呜呜

**Reply by zi4dbek** [reply]
就这么升级：直接下载最新版本替换就行

**Comment by StartMoney** [comment]
replit中用mysql做为存储时，配置环境变量不起作用。程序还是默认生成data目录和 data.db的文本数据库。
